src/performance/navigation.ts

```diff
@@ -0,0 +1,27 @@
+import { Event } from "@clarity-types/data";
+import { NavigationData } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let data: NavigationData = null;
+
+export function reset(): void {
+    data = null;
+}
+
+export function compute(entry: PerformanceNavigationTiming): void {
+    data = {
+        fetchStart: Math.round(entry.fetchStart),
+        connectStart: Math.round(entry.connectStart),
+        connectEnd: Math.round(entry.connectEnd),
+        requestStart: Math.round(entry.requestStart),
+        responseStart: Math.round(entry.responseStart),
+        responseEnd: Math.round(entry.responseEnd),
+        domInteractive: Math.round(entry.domInteractive),
```

Comment:
Add domContentLoadedEventStart;, domContentLoadedEventEnd;

Reply:
I was hoping to add these metrics as we have a use for them. Not add everything in first go. What are your thoughts? I've added redirectCount and loadEventStart for completeness. I was in two minds about this. 

---
In reply to: [347587449](https://github.com/microsoft/clarity-js/pull/160#discussion_r347587449) [](ancestors = 347587449)